### ISSUE TITLE
Minor inconsistency with `Reconciliation Query`'s `type` field

### ISSUE THREAD

**Issue by derenrich** [reconciliation-api/specs#152]
The [spec ](https://www.w3.org/community/reports/reconciliation/CG-FINAL-specs-0.2-20230410/#structure-of-a-reconciliation-query) says that the `type` field should be 

>An array of [types](https://www.w3.org/community/reports/reconciliation/CG-FINAL-specs-0.2-20230410/#dfn-type). Supplying such types allows users to restrict the search to entities which bear those types

But then the example shows

```json
{
    "type": "DifferentiatedPerson"
}
```

Looking at the json schema it actually says that type is

> Either a single type identifier or a list of type identifiers

which I guess sounds ok but it still doesn't match the linked definition of type in [section 2.2 ](https://www.w3.org/community/reports/reconciliation/CG-FINAL-specs-0.2-20230410/#types) which says

>A type represents a category of [entities](https://www.w3.org/community/reports/reconciliation/CG-FINAL-specs-0.2-20230410/#dfn-entity). It comprises the following fields:
>id
>    an identifier, which is a non-empty string. This identifier must be unique among all types;
> name
>    a human-readable name, which is a non-empty string;
> broader
 >   an optional array 

which I think is the definition of the type object when its returned

**Comment by derenrich** [comment]
Also the definition of `type` in section 2.2 refers to a field of it called `broader` but that appears to never be referenced in the jsonschema? The json schema looks like:

```json
                    {
                      "type": "object",
                      "description": "A type can be given by id and name",
                      "properties": {
                        "id": {
                          "type": "string"
                        },
                        "name": {
                          "type": "string"
                        }
                      },
                      "required": [
                        "id"
                      ]
                    },
```

**Comment by wetneb** [comment]
This is indeed something that we have been working on in the new version of the specification: see https://github.com/reconciliation-api/specs/issues/109.

**Comment by fsteeg** [comment]
Resolved with #109 / #115, closing.
